src-docs/src/views/form_layouts/form_rows.js

```diff
@@ -59,6 +59,13 @@ export default () => {
         <EuiFieldText name="first" />
       </EuiFormRow>
 
+      <EuiFormRow
+        label="Text field"
+        helpText="This is a disabled input field."
+        isDisabled>
```

Comment:
We don't need to demo `helpText` again and having a unique label for every input is better for accessibility (even though this is a demo page, we still want to have good examples). 

```suggestion
        label="Disabled text field"
        isDisabled>
```

Reply:
I totally missed that. Thanks for changing that!